Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -23,10 +23,10 @@ Oppia relies on a number of programs and third-party libraries. Many of these li
 sudo apt-get update
 ```
 
-2. Make sure you have curl (used to download third-party libraries), setuptools (needed for installing coverage, which checks test coverage for the Python code), git (which allows you to store the source in version control), python-dev (which is used for the numpy installation), python-pip (which is also used for the numpy installation), pyyaml (which is used to parse YAML files) and libbz2-dev (used by Apache Beam):
+2. Make sure you have curl (used to download third-party libraries), git (which allows you to store the source in version control), unzip (used for unpacking zip files) and openjdk-11-jre (needed for GCP to work):
 
 ```
-sudo apt-get install python3-pip python3-setuptools curl openjdk-8-jre git python3-dev python3-yaml python3-matplotlib unzip libbz2-dev
+sudo apt-get install curl git unzip openjdk-11-jre
```

Comment:
I think running sudo update-alternatives --config java command should be put here in case the dev has multiple versions and wants to make it default.

Reply:
@seanlip Maybe the below note should be moved here since its right after java 11 installed?

Reply:
Thanks @Helper2020, done! PTAL. I also merged the first two steps.